src/ResourceManager/Sql/Commands.Sql/Replication/Cmdlet/NewAzureSqlDatabaseSecondary.cs

```diff
@@ -30,6 +32,9 @@ namespace Microsoft.Azure.Commands.Sql.Replication.Cmdlet
         ConfirmImpact = ConfirmImpact.Low, SupportsShouldProcess = true)]
     public class NewAzureSqlDatabaseSecondary : AzureSqlDatabaseSecondaryCmdletBase
     {
+        private const string DtuDatabaseParameterSet = "DtuBasedDatabase";
```

Comment:
@payiAzure make sure to add a default parameter set to this cmdlet that points at the previous parameter set

Reply:
@cormacpayne Thanks for catching this. Add the default parameter set for NewAzureSqlDatabaseSecondary and NewAzureSqlDatabaseCopy